specification/app/resource-manager/Microsoft.App/preview/2023-05-01-preview/Builders.json

```diff
@@ -0,0 +1,487 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "builderName",
+            "in": "path",
+            "required": true,
+            "description": "The name of the builder.",
+            "minLength": 2,
+            "maxLength": 32,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Resource deleted successfully."
+          },
+          "202": {
+            "description": "Resource deletion accepted.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Resource deleted successfully."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Builders_Delete_0": {
+            "$ref": "./examples/Builders_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "BuilderProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning state of a builder resource.",
+          "readOnly": true
+        },
+        "environmentId": {
+          "type": "string",
+          "description": "Resource ID of the container apps environment that the builder is associated with."
+        },
+        "buildComputeId": {
+          "type": "string",
+          "description": "Resource ID of the compute that the builds will be performed on.",
+          "readOnly": true
+        },
+        "patchComputeId": {
+          "type": "string",
+          "description": "Resource ID of the compute that the patching will be performed on.",
+          "readOnly": true
+        }
+      },
+      "description": "The builder properties.",
+      "required": [
+        "environmentId"
+      ]
+    },
+    "BuilderResource": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/BuilderProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v4/managedidentity.json#/definitions/ManagedServiceIdentity",
```

Comment:
Please change to the same ref path
https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2023-05-01-preview/specification/app/resource-manager/Microsoft.App/preview/2023-05-01-preview/Jobs.json#L732
https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2023-05-01-preview/specification/app/resource-manager/Microsoft.App/preview/2023-05-01-preview/ContainerApps.json#L555